plugins/fingerquoter.go

```diff
@@ -29,6 +28,10 @@ type FingerQuoter struct {
 	frequency       int
 }
 
+// Regular expression to find candidate words. They must be at least 5 characters long
+// and can include any word character (include hyphen and underscore)
+var candidateWordsRegexp = regexp.MustCompile("(?:\\s|\\A)([\\w-]{5,})")
```

Comment:
Playing with the `regexp` to do proper finding of candidate words: https://play.golang.org/p/gvO64ikYGpi